components/ingredients_comp.js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react"
+import EntryDropdown from "./entry_dropdown_comp"
+import KitchenList from "./kitchen_list_comp"
+import Stack from "@mui/material/Stack"
+
+export default function Ingredients(props) {
+	//const ingredients = props.ingredients
+
+	const [ myIngredients, setMyIngredients ] = useState(props.myIngredients)
```

Comment:
Are you supposed to have two versions of myIngredients like this? seems like props.myIngredients should just be used instead

Reply:
agreed, I realized this will cause problems when I actually try to implement sending new/deleting ingredients in a post/delete request. For now I'll merge it